beacon-chain/core/helpers/committee_test.go

```diff
@@ -175,51 +175,6 @@ func TestComputeCommittee_WithoutCache(t *testing.T) {
 	}
 }
 
-func TestComputeCommittee_WithCache(t *testing.T) {
```

Comment:
wouldn't you want to add the same tests when you add in the new cache? maybe just comment ?

Reply:
It's already tested in in the cache package, since now we are using a feature flag, this was just testing that the feature flag should work